<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="importExcelButton">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>600</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Import</string>
   </property>
  </widget>
  <widget class="QListView" name="listView_2">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>50</y>
     <width>241</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="supplierList">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>50</y>
     <width>241</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="itemList">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>270</y>
     <width>241</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="customer_location_edit">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>70</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>70</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>ที่ตั้งลูกค้า</string>
   </property>
  </widget>
  <widget class="QLabel" name="supplier_label">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>110</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ชื่อซัพพลายเออร์ :</string>
   </property>
  </widget>
  <widget class="QLabel" name="supplier_location_label">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>130</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ตำแหน่ง :</string>
   </property>
  </widget>
  <widget class="QLabel" name="supply_label">
   <property name="geometry">
    <rect>
     <x>1090</x>
     <y>150</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>สินค้า  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="transport_cost_label">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>170</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ราคาขนส่ง :</string>
   </property>
  </widget>
  <widget class="QLabel" name="supply_cost_label">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>190</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>ต้นทุนรวม  :</string>
   </property>
  </widget>
  <widget class="QWidget" name="map_widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>721</width>
     <height>531</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>600</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
